examples/utils/example_client.rs

```diff
@@ -63,20 +63,19 @@ impl ExampleClient {
                         break 'outer;
                     }
                     Event::Terminate => {
-                        println!("Client failed to connect to the network. Restarting.");
-                        thread::sleep(Duration::from_secs(5));
-                        break;
+                        println!("Client failed to connect to the network.");
+                        return None;
                     }
                     _ => (),
                 }
             }
         }
 
-        ExampleClient {
+        Some(ExampleClient {
             routing_client: routing_client,
             receiver: receiver,
             full_id: full_id,
-        }
+        })
```

Comment:
those option just to avoid hanging when execute the examples
if ci scripts only checks the compilation, will be ok to change back.